client/src/pages/Lobby.tsx

```
@@ -385,14 +370,42 @@ export default function Lobby() {
-                                                  placeholder:font-medium text-[#4F5666] font-medium"
-                                placeholder="0x..."
-                                onChange={(e) => setPlayer2(e.target.value)}
-                              />
+                              {playWith === "Human" && (
+                                <>
+                                  <input
+                                    className="p-2.5 w-72 rounded-xl border border-[#1D212B] bg-transparent outline-none placeholder:text-[#4F5666] placeholder:font-medium text-[#4F5666] font-medium"
+                                    placeholder="0x..."
+                                    onChange={(e) => setPlayer2(e.target.value)}
+                                  />
+                                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)